infrastructure/testdriver/virtual_authenticator.html

```diff
@@ -41,7 +41,7 @@
 }, "Can create an authenticator");
 
 promise_test(async t => {
-  return test_driver.add_credential(authenticator_id, credential);
+  return await test_driver.add_credential(authenticator_id, credential);
```

Comment:
I"m not sure this actually changes anything?

Reply:
Sorry this sneaked in. I'll remove it.